app/views/shared/_navigation_bar.html.erb

```diff
@@ -0,0 +1,40 @@
+<nav class="navbar navbar-expand-lg navbar-light">
+  <div class="container">
+    <%= link_to "Home", root_path , class: "navbar-brand fs-3" %>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <%= link_to "Foods", foods_path ,class:"nav-link fs-5" %>
+        </li>
+        <% unless current_user.nil? %>
+          <li class="nav-item">
+            <%= link_to "My Recipes", recipes_path ,class:"nav-link fs-5" %>
+          </li>
+        <% end %>
+        <% unless current_user.nil? %>
+          <li class="nav-item">
+            <%= link_to "My Inventories", inventories_path ,class:"nav-link fs-5" %>
+          </li>
+        <% end %>
+        <li class="nav-item">
+          <%= link_to "Public Recipes", public_recipes_path ,class:"nav-link fs-5" %>
+        </li>
+        <% if current_user.nil? %>
+          <li class="nav-item">
+            <%= link_to "Log in", new_user_session_path ,class:"nav-link fs-5" %>
+          </li>
+        <% else %>
+         <li class="nav-item">
+            <%= button_to 'Logout', destroy_user_session_path, method: :delete ,class:"nav-link fs-5 logout" %>
+          </li>
+        <% end %>
+      </ul>
+    </div>
+    <% unless current_user.nil? %>
+      <span id="username-navbar">User: <%= current_user.name %></span>
+    <% end %>
+  </div>
+</nav>
```

Comment:
- [ ] There seems to be an issue with your bootstrap javascript that makes me unable to open your navbar in mobile or tablet versions. I would recommend you fix it.

  ![navbar-1](https://user-images.githubusercontent.com/36542986/175531816-efd80753-a6eb-4c32-aa89-70cba4a57793.jpg)

Reply:
@TedLivist Thanks for pointing that out, but we don't have any requirements about mobile responsivity, I've had issues with bootstrap so the menu isn't working, for now, I'll try to fix it but I want to get the project approved first as it is an optional feature.

Reply:
Oh yeah. I intended that to be optional. But forgot to add the tag

Reply:
Thanks a lot for correcting that I appreciate the suggestion, I'll make sure to implement it.